### ISSUE TITLE
[Feature Request] Combine various timestep sensors into a single sensor with a list of attributes

### ISSUE THREAD

**Issue by ajayjohn** [r-renato/ha-climacell-weather#15]
If possible, could you please create a single entity for hourly or daily forecast and providing all the values related to it as an array of attributes available on it? I am not sure if this something the custom component takes care of or if it is handled by HA itself. But, this is how all the weather type entities seem to work.

Attached here is an example of how the NOAA component work. If you notice here, there is only one entity for the daily forecast. However, the entity holds the attributes needed for each and every forecast timestep. I feel like the Climacell component, as it adds more and more attributes, can make use of such a structure to avoid too many new entities from being created.

![image](https://user-images.githubusercontent.com/1575879/86994461-06a00300-c16c-11ea-8992-27d6d30e2287.png)
![image](https://user-images.githubusercontent.com/1575879/86994471-0a338a00-c16c-11ea-9547-b1804d64a527.png)

**Comment by r-renato** [comment]
What is the name of the integration?

**Comment by ddellspe** [comment]
I believe that's leveraging the [`weather` entity](https://developers.home-assistant.io/docs/core/entity/weather) from homeassistant core.  I've been looking into it myself and was considering taking some of the climacell work that you've done to provide some consistent weather entities and potentially writing a frontend that can provide more than just temperature forecast, etc.

**Comment by ajayjohn** [comment]
@r-renato As @ddellspe mentioned, any weather type entity (https://www.home-assistant.io/integrations/#weather) would typically behave this way. The ones I have in the screenshots are https://www.home-assistant.io/integrations/weather.darksky/ and https://www.home-assistant.io/integrations/nws/.

@ddellspe It would be awesome something like that would be available in HA!

**Comment by K-Money** [comment]
I was just about to request the same thing myself. Glad there's already one on here. 

The weather platform would also allow the usage of other lovelace cards since they all require a weather entity to function rather than individual sensors. e.g. https://www.home-assistant.io/lovelace/weather-forecast/ . 

The existing Weather Conditions Card is great for pairing with this repo but may not aesthetically work in all cases for the end user. Adding the weather entity would allow the user more options.

**Comment by cerebrate** [comment]
Yes, indeed. (Although I would prefer that this was in addition to, rather than instead of, the current individual sensors, because it's handy to be able to use them in various places without needing a template to break out the specific attributes.)

As a side note, up until today I've been using the Weather Template integration on HACS (https://github.com/xannor/hass_weather_template) to compose a weather.climacell from the individual sensors, but it looks like 0.115.1's changes to the template platform have broken this option.

**Comment by Chaotic** [comment]
> Yes, indeed. (Although I would prefer that this was in addition to, rather than instead of, the current individual sensors, because it's handy to be able to use them in various places without needing a template to break out the specific attributes.)
> 
> As a side note, up until today I've been using the Weather Template integration on HACS (https://github.com/xannor/hass_weather_template) to compose a weather.climacell from the individual sensors, but it looks like 0.115.1's changes to the template platform have broken this option.

Any chance you could share your template config till this gets updated?

**Comment by cerebrate** [comment]
Sorry, I'm afraid I deleted it shortly after 0.115 came out because of that issue I was having with the template platform. I'm now just using https://github.com/r-renato/ha-card-weather-conditions instead.
